library/test/src/cli.rs

```diff
@@ -51,12 +51,11 @@ fn optgroups() -> getopts::Options {
         .optflag("", "test", "Run tests and not benchmarks")
         .optflag("", "bench", "Run benchmarks instead of tests")
         .optflag("", "list", "List all tests and benchmarks")
-        .optflag("h", "help", "Display this message (longer with --help)")
+        .optflag("h", "help", "Display this message")
         .optopt(
             "",
             "logfile",
-            "Write logs to the specified file instead \
-             of stdout",
+            "Write logs to the specified file",
```

Comment:
Why was this change made?

Reply:
I think it is incorrect.  Normal output continues to go to stdout with `--logfile`.  I discovered some of the help text was incorrect while writing the chapter.

Reply:
> Normal output continues to go to stdout with `--logfile`.

What is "normal output"? Do you meant that with `--logfile` logs still go to stdout, but they also go to the log file?

Reply:
Normal output is the output the harness normally displays, such as:

```
 test it_works ... ok 
```

With `--logfile`, it writes a specific format to the given file, which looks like:

```
ok it_works
```

I was reluctant to document the specific format, since I don't know if it is intended to be stable or relied upon. It was added without much detail in #2127, and I'm not sure anyone actually uses it (I haven't seen it used before).